source/scenes/Registration/components/Form/Panes/selectOptions.js

```diff
@@ -25,6 +25,7 @@ export const diet = [
   { text: 'Vegetarian', value: 'VEGETARIAN' },
   { text: 'Gluten Free', value: 'NOGLUTEN' },
   { text: 'Nut Allergy', value: 'NOPEANUT' },
+  { text: 'Other', value: 'OTHER' },
```

Comment:
An "Other" allergy doesn't really help us, since we can't plan meals around that.

Reply:
agreed